### DISCUSSION TITLE
The close button is no longer appearing on random tabs...setting or bug?

### DISCUSSION THREAD

**Discussion by paulescobar** [piroor/treestyletab#3365] [Q&A] [answered]
![image](https://github.com/piroor/treestyletab/assets/22924037/875de20e-ab9f-4fa5-9983-2f91fb956ad0)

Before, the close button would appear over all tabs.
Now all of a sudden, it seems to randomly appear over some tabs - but not others.

I'm very new to FireFox and this add-on. So I'm not sure if it's a settings problem or perhaps a bug from a recent update?

**Comment by piroor** [comment]
Do you use any custom CSS in TST's options? It may need to be updated for 3.9.16.

**Reply by paulescobar** [reply]
I was testing random tweaks in TST settings a few days ago.
But as far as I know, I deleted them and assumed the default was there.

This is what's currently in my "Extra Style Rules" section of TST settings:
```
/* Show title of unread tabs with red and italic font */
/*
:root.sidebar tab-item.unread .label-content {
  color: red !important;
  font-style: italic !important;
}
*/

/* Add private browsing indicator per tab */
/*
:root.sidebar tab-item.private-browsing tab-label:before {
  content: "🕶";
}
*/
```

I also have the following in my FireFox's "userchrome.css" file:
```
/* Limit the width of the Flexible Space on all toolbars to ? pixels */
toolbar toolbarspring{
  -moz-box-flex: 40 !important;
  min-width: 5px !important;
  max-width: 10px !important;
}
```
Could any of these have an influence on this problem?

**Reply by paulescobar** [reply]
Just an additional note, I notice the close button goes missing on every tab that has a long title. The tabs that have short titles all show the close button.

**Reply by paulescobar** [reply, marked as answer]
Hi @piroor I figured out what's causing the problem!
It's not "CSS" code.

I had the extension "TST Tab Search" installed (it adds a tab search box at the bottom of the TST sidebar).
When I disabled the extension, all the close buttons returned and everything looks normal.

It's strange because the search bar is all the way at the bottom.
Not sure why it would interfere with the close icons at the side.

![image](https://github.com/piroor/treestyletab/assets/22924037/dc1ca720-7fc2-4fa0-af64-cefbceded0ab)
![image](https://github.com/piroor/treestyletab/assets/22924037/381a08c8-2b75-4284-84ff-4a304dedc3bb)

**Reply by dstd** [reply]
Related issue: https://github.com/NiklasGollenstede/tst-search/issues/29
